src/main/java/org/prebid/server/floors/BasicPriceFloorProcessor.java

```diff
@@ -259,7 +259,7 @@ private BidRequest updateBidRequestWithFloors(BidRequest bidRequest,
     }
 
     private static boolean shouldSkipFloors(Integer skipRate) {
-        return skipRate != null && ThreadLocalRandom.current().nextInt(SKIP_RATE_MAX) < skipRate;
+        return skipRate != null && ThreadLocalRandom.current().nextInt(SKIP_RATE_MAX) <= skipRate;
```

Comment:
This change doesn't make sense because random can return values ​​in [0, 99] range

Reply:
Also, there is chance that with zero skipRate it will return `true`.

Reply:
when skip rate is 99 it will always be skipped

Reply:
nope, 99 < 99 = false

Reply:
yeah, brain lag) will close the PR